frontend/rust-lib/flowy-user/src/event_handler.rs

```diff
@@ -652,3 +652,25 @@ pub async fn update_workspace_member_handler(
     .await?;
   Ok(())
 }
+
+#[tracing::instrument(level = "debug", skip_all, err)]
+pub async fn add_workspace_handler(
```

Comment:
Does "add" here mean "join" or "create"?

Reply:
means creating a brand new workspace, I'll rename it to `create_workspace_handler`